docs/essentials/permissions.md

```diff
@@ -85,7 +85,7 @@ Icon Guide:
 | StorageRead | ![Android supported](~/media/shared/yes.png "Android supported") | ![iOS not supported](~/media/shared/no.png "iOS not supported") | ![UWP not supported](~/media/shared/no.png "UWP not supported") | ![watchOS not supported](~/media/shared/no.png "watchOS not supported") | ![tvOS not supported](~/media/shared/no.png "tvOS not supported") | ![Tizen not supported](~/media/shared/no.png "Tizen not supported") |
 | StorageWrite | ![Android supported](~/media/shared/yes.png "Android supported") | ![iOS not supported](~/media/shared/no.png "iOS not supported") | ![UWP not supported](~/media/shared/no.png "UWP not supported") | ![watchOS not supported](~/media/shared/no.png "watchOS not supported") | ![tvOS not supported](~/media/shared/no.png "tvOS not supported") | ![Tizen not supported](~/media/shared/no.png "Tizen not supported") |
 
-If a permission is marked with a ❌ it will always return `Granted` when checked or requested.
+If a permission is marked with a ![❌](~/media/shared/no.png "❌") it will always return `Granted` when checked or requested.
```

Comment:
an emoji is not appropriate for alt or title text on an image. these values are used for accessibility (screen readers, etc).

Reply:
Thank you for your advice 😄